go/codeql-tools/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@rules_pkg//pkg:mappings.bzl", "pkg_attributes", "pkg_filegroup", "pkg_files")
+
+pkg_files(
+    name = "executables",
+    srcs = glob(["*.sh"]),
```

Comment:
What about the Windows-specific `.cmd` files?

Reply:
Executable permission bits are a posix thing only. Windows doesn't need them to execute files (and indeed the `.cmd` files do not currently have the bit set)

Reply:
Ah sorry, I think I misunderstood what this part is doing. I had assumed that this declaration was about _including_ these files in a distribution and that `attributes` would only apply on systems where that made sense (hence my comment isn't on that line). I can see now that the next `pkg_files` declaration will include the `.cmd` files. 

It may be worth making this more obvious in the naming, since this being called "executables" and the other declaration being called "non-executables" made it look like omitting the `.cmd` files was an oversight.

Reply:
good point!

Reply:
One follow-up question here: would it make sense to tease the `.cmd` files out into their own group and then only bundle the `.sh` files on Linux/macOS and the `.cmd` files only on Windows?

Reply:
That doesn't match what we do in the other languages - we ship all non-executables in a platform-independent language pack (including platform-specific scripts), and only the binaries are separated into the per-platform directories in the dist, and shipped separately.

We could change this, but right now, we resolve these files in the shared folder, and if we change this, we should change this across all languages, to not be inconsistent.
I'd say the cost/benefit trade-off of this change is not worth it.

Reply:
By the way, if we port our internal packaging rules here in `codeql` or in another `codeql-bazel-rules` open source repo, I'd solve this more gracefully, maybe with an attribute map similar to how `renames` works.